web/common/fetchers.ts

```diff
@@ -0,0 +1,41 @@
+import { API_URL } from './constants';
+ 
+export const saveSession = async (dehydratedState: string) => {
+  await fetch(API_URL + '/api/session/save', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ dehydratedState }),
+  });
+};
+ 
+export const destroySession = async () => {
+  try {
+    await fetch(API_URL + '/api/session/destroy', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: null,
+    });
+  } catch (e) {
+    console.log('ERROR:', e);
+  }
+};
+
+export const createUser = async (formData: Object) => {
```

Comment:
Q: were is this method used or why would it be needed?

Reply:
Didn't need it (yet). I can remove it until we need it again